packages/forms/tailwind/default-options.js

```diff
@@ -221,14 +221,22 @@ function defaultOptions({ config }) {
       },
 
       parts: {
-        labelContainer: {
-          lineHeight: defaultTheme.lineHeight.tight,
-          display: 'flex',
-          alignItems: 'flex-start'
-        },
         inputContainer: {
+          lineHeight: defaultTheme.lineHeight.tight,
```

Comment:
`lineHeight: defaultTheme.lineHeight.tight` 

not sure if this should be on the label or  here